cluster-autoscaler/cloudprovider/gce/gce_url.go

```diff
@@ -73,6 +73,11 @@ func GenerateMigUrl(ref GceRef) string {
 	return fmt.Sprintf(migUrlTemplate, ref.Project, ref.Zone, ref.Name)
 }
 
+// IsInstanceTemplateRegional determines whether or not an instance template is regional based on the url
+func IsInstanceTemplateRegional(templateUrl string) (bool, error) {
+	return regexp.MatchString("(/projects/.*[A-Za-z0-9]+.*/regions/)", templateUrl)
```

Comment:
Failing tests: We need to add the regexp import as it doesn't exist in this file.

Reply:
Good catch, thanks! Added the missing import.